memcached/init.lua

```diff
@@ -82,6 +82,10 @@ memcached_get_stat (struct memcached_service *);
 struct memcached_service *
 memcached_create(const char *, uint32_t);
 
+void memcached_slab_arena_create();
```

Comment:
AFAIC, you didn't cover module deinitialization: why shouldn't we care about module unloading from Lua? It seems there are [ways](https://www.freelists.org/post/luajit/BUG-Assertion-failures-when-unloading-and-reloading-the-ffi-package,1) to do this in runtime, so we should care about this.

Perhaps adding a routine with `__attribute__((__destructor__))` would suffice?